S100-Permit-Service-Open-Api-Sepc-v0.1.yaml

```diff
@@ -0,0 +1,162 @@
+openapi: 3.0.1
+info:
+  title: UKHO S100 Permit Service API
+  description: |
+    The UKHO S100 Permit Service API.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+  
+externalDocs:
+  url: https://github.com/UKHO/s-100-permit-service/tree/main
+servers:
+  - url: https://permit-service-url
+    description: Proposed URL
+
+security:
+  - jwtBearerAuth: []
+  
+paths:
+  /permits/{licenceId}:
+    get:
+      description: |
+        This service uses the S-100 Part 15 data protection scheme to generate unsigned PERMIT.XML files for the respective User Permit Numbers (UPNs) for the requested licence and returns the compressed zip file containing all these PERMIT.XML files.
```

Comment:
This API will generate `signed` permits, though it is outside of WP scope.

Reply:
Updated.